### ISSUE TITLE
Crazy names in the output of mrpet/std_template

### ISSUE THREAD

**Issue by IKDRipp** [Neurita/pypes#43]
My dear Alex,

Please clean the list of issues, you are such a lazy bastard. ;)

Also wtf is this file: pet_recon_grptemplate.nii doing in the std_template output. Could you check the names of these files, they are horrible and I do not know if there is any human being able to understand them. ;)

Have a look at this:
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session2_nii/.../mrpet/std_template/pet_recon_grptemplate.nii
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session2_nii/.../mrpet/std_template/wpet_recon_rbv_pvc.nii
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session2_nii.../mrpet/std_template/wpet_recon_rbv_pvc_intnormed.nii
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session2_nii/.../mrpet/std_template/wtissues_brain_mask.nii
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session2_nii/.../mrpet/std_template/y_rmHead_MPRAGE_highContrast_corrected.nii

Also:

/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session1_nii/.../mrpet/tissues/csf_Head_ep2d_pace_rest.nii
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session1_nii/.../mrpet/tissues/gm_Head_ep2d_pace_rest.nii
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session1_nii/.../mrpet/tissues/wm_Head_ep2d_pace_rest.nii

**Comment by alexsavio** [comment]
Hi @IKDRipp,

Thanks for your passive-aggressive messages, you're such a kind person.

Anyway, after writing myself this issue and thinking about it without your breath on the back of my neck, I will go by parts:
`pet_recon_grptemplate.nii.gz` is the `pet_recon` in `grptemplate` space, as you can see here: https://github.com/Neurita/pypes/blob/master/neuro_pypes/pet/grouptemplate.py#L149

It is true that the other files have a weird name, they start with `w`. This is the SPM convention, and my inability to catch the correct regular expression to rename these files. I will have a look.

**Comment by IKDRipp** [comment]
Dear @alexsavio, 
a good working atmosphere is crucial,  I am very glad you think I am a kind person :innocent:

So the grptemplate space is also MNI spaced, right!? As it relies on the SPM PET (15 O) template in a previous step!? 
Also interesting and worth implementing in the future:

https://www.ncbi.nlm.nih.gov/pubmed/24952892

**Comment by alexsavio** [comment]
Yes, it is in MNI, because the group template is built with PET images warped to MNI.

I love it when you give me more work to do in such a subtle way. The reference is interesting, maybe you could create a new issue for that, just so you do something and we don't lose such a nice suggestion through our polite conversations. I wonder if they made the template public available. 😉

**Comment by alexsavio** [comment]
Dear @IKDRipp,

Talking about:
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session1_nii/.../mrpet/tissues/csf_Head_ep2d_pace_rest.nii
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session1_nii/.../mrpet/tissues/gm_Head_ep2d_pace_rest.nii
/media/iripp/seagate_external/Projects/MRPET15_trial/out/Session1_nii/.../mrpet/tissues/wm_Head_ep2d_pace_rest.nii

Could you please give me a list of the files in the rest/tissues folder? I think the rest-fMRI regular expression (regex) is stepping over the PET regex. To improve the both, I need to know what I should look for in the rest-fMRI folder. Thanks!

**Comment by alexsavio** [comment]
Hi, sorry, no need. I got it!
